tests/units/test_catalog.py

```diff
@@ -200,6 +200,18 @@
 ]
 
 
+def test_merge_catalogs() -> None:
+    """Test the merge_catalogs function."""
+    # Load catalogs of different sizes
+    small_catalog = AntaCatalog.parse(DATA_DIR / "test_catalog.yml")
+    medium_catalog = AntaCatalog.parse(DATA_DIR / "test_catalog_medium.yml")
+    tagged_catalog = AntaCatalog.parse(DATA_DIR / "test_catalog_with_tags.yml")
+
+    # Merge the catalogs and check the number of tests
+    final_catalog = merge_catalogs([small_catalog, medium_catalog, tagged_catalog])
+    assert len(final_catalog.tests) == 240
```

Comment:
to make it more robust you could have `== len(small) + len(medium) + len(tagged)` this way if we ever change them the test won't fail because of hardcoded `240`

Reply:
Fixed in 28e3fa985af88c21c1403185c5a07eb1cdd61a24